packages/cursorless-engine/src/languages/TreeSitterQuery/isContainedInErrorNode.ts

```diff
@@ -0,0 +1,19 @@
+import type { SyntaxNode } from "web-tree-sitter";
+
+/**
+ * Determines whether the given node or one of its ancestors is an error node
+ * @param node The node to check
+ * @returns True if the given node is contained in an error node
+ */
+export function isContainedInErrorNode(node: SyntaxNode) {
+  let currentNode: SyntaxNode | null = node.parent;
```

Comment:
why do you start with the parent? Why not start with the node itself

Reply:
The dangers of copilot. Fixed now.